dev/tools/localization/gen_l10n.dart

```diff
@@ -37,6 +37,23 @@ import 'messages_all.dart';
 /// );
 /// ```
 ///
+/// ## Update pubspec.yaml
+///
+/// Please make sure to update your pubspec.yaml to include the following
+/// packages:
+///
+/// ```
+/// dependencies:
+///   # rest of dependencies
```

Comment:
Including "rest of dependencies" twice doesn't look right. Maybe just include the second one.